services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
+
+from db.models import Ticket, Order, User, MovieSession
+
+
+def create_order(tickets: list[Ticket], username: str,
+                 date: datetime = None) -> None:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
Consider handling the case where the User with the given 'username' does not exist, which would raise a 'DoesNotExist' exception. You might want to catch this exception and handle it appropriately.